generate/src/Write/Utils.hs

```diff
@@ -34,7 +34,9 @@ docComment docSymbol c = case comment c of
                  _ -> error "'comment' didn't return a comment"
 
 postdocComment :: String -> String
-postdocComment = docComment '^'
+postdocComment c = case docComment '^' c of
+  "" -> ""
+  s -> " " ++ s
```

Comment:
I think that this may be a little neater if it were to be split into a utility function `prependGap :: String -> String` and the original version of `postdocComment`